### ISSUE TITLE
Send marker for entries

### ISSUE THREAD

**Issue by TuxUbuntu** [dignifiedquire/async-tar#3]
Send market isn't implement for entries.

```rust
use tokio::fs::File;
use tokio_tar::Archive;
use futures::prelude::*;
use tokio::prelude::*;

#[tokio::main]
async fn main() {
	tokio::spawn(async {
		let file = File::open("/dev/null").await.unwrap();
		let mut a = Archive::new(file);
		let mut entries = a.entries().unwrap();
		entries.next().await;
	});
}
```
Dependencies:

```toml
tokio-tar = "0.1.0" # or async-tar = "0.1.1"
futures = "0.3.4"
tokio = { version = "0.2.13", features = ["full"] }
```

**Comment by tailhook** [comment]
I have the same issue with `append` and `append_data`. Is it possible to add Send trait on methods?

**Comment by dignifiedquire** [comment]
It is possible, but requires adding a Mutex, where there currently is none. I will think a bit on it

**Comment by zkat** [comment]
The Archive isn't either, and I think this is what's causing some trouble I've run into :( https://github.com/dignifiedquire/async-tar/blob/master/src/archive.rs#L45

**Comment by dignifiedquire** [comment]
Should be fixed in the new `0.2` release
